src/test/java/step3/domain/LottoStatisticsTest.java

```diff
@@ -0,0 +1,99 @@
+package step3.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step3.domain.lotto.Lotto;
+import step3.domain.lotto.LottoMachine;
+import step3.domain.lotto.LottoNumber;
+import step3.domain.lotto.LottoRank;
+import step3.domain.lotto.LottoStatistics;
+import step3.domain.lotto.Profit;
+import step3.domain.lotto.ResultOfLottos;
+import step3.domain.lotto.WinOfLotto;
+import step3.domain.money.Cache;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class LottoStatisticsTest {
+
+    private WinOfLotto givenWinOfLotto;
+
+    @BeforeEach
+    void setUp() {
+        Lotto winOfLottoNumbers = Lotto.create(Arrays
+            .asList(new LottoNumber(1), new LottoNumber(2), new LottoNumber(3), new LottoNumber(4), new LottoNumber(5), new LottoNumber(6)));
```

Comment:
```suggestion
            .asList(1, 2, 3, 4, 5, 6);
```
정적 팩토리 메서드를 사용하니 요런 방식으로 간단하게 생성하는 메서드를 지원해도 좋을 것 같아요~!